### DISCUSSION TITLE
bpf_helper_defs.h: error: unknown type name

### DISCUSSION THREAD

**Discussion by bithamr** [cilium/ebpf#813] [Ideas]
if you're [...] getting bpf_helper_defs.h:77:83: error: unknown type name: '__u64' you need to make sure #include <linux/types.h> is your first include.

**Comment by lmb** [comment]
@bithamr discovered the fix to his problem himself:

> if you're [...] getting `bpf_helper_defs.h:77:83: error: unknown type name: '__u64'` you need to make sure `#include <linux/types.h>` is your first include.
